x-pack/plugins/session_view/public/components/ProcessTreeNode/styles.ts

```diff
@@ -138,7 +136,7 @@ export const useStyles = ({ depth }: StylesDeps) => {
       workingDir,
       userEnteredIcon,
     };
-  }, [depth, euiTheme]);
+  }, [depth, colors, border, font, size]);
```

Comment:
nitpick: do we need these as dependency? Does the `euiTheme` reference change when it's properties change?

Reply:
hey, good catch, the reference does not necessarily change, it was only an Eslint complaint, so I moved the variables declaration, so we can keep the dependency tree cleaner